File: CmdMents/God/AcknowledgeGodAsHoly.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.God
{
    class AcknowledgeGodAsHoly : KnowGodExists
    {
        public AcknowledgeGodAsHoly()
        {
            base.AlternateText = null;
            base.Book = CommandmentBook.Leviticus;
            base.CanBeCarriedOutToday = true;
            base.CanBeCarriedOutOnlyInIsrael = false;
            base.Chapter = 22;
            base.CommandmentType = CommandmentType.Positive;
            base.FollowedByChristians = CommandmentObedience.Obeyed;
            base.FollowedByMessianics = CommandmentObedience.Obeyed;
            base.FollowedByObservantJews = CommandmentObedience.Obeyed;
            base.Number = 6;
            base.ShortSummary = "Acknowledge God as holy.";
            base.Text = "I must be acknowledged as holy by the Israelites.";
            base.Verse = 32;
        }
    }
}

File: EtzMitzvot/Commandments/God/acknowledgeGodAsHoly.ts
///<reference path='../commandmentBase.ts' />

class AcknowledgeGodAsHoly extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Acknowledge God as holy.";
        this.text = "I must be acknowledged as holy by the Israelites.";
        this.book = TorahBook.Leviticus;
        this.chapter = 22;
        this.verse = 32;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.Binding;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 6;

        this.children = [
            new FearGod(), new BlessGodAfterEating(), new DontCurseGod(), new DontTemptGod()
        ];
    }
}